backend/src/test/java/pl/edu/agh/api/controller/ActorsControllerTest.java

```diff
@@ -18,17 +22,14 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-import pl.edu.agh.model.Fighter;
-import pl.edu.agh.service.FighterService;
-import java.util.List;
-import java.util.UUID;
-
 
 @WebMvcTest(ActorsController.class)
 public class ActorsControllerTest {
-    @Autowired private MockMvc mockMvc;
-    @MockBean private FighterService fighterService;
     private final ObjectMapper objectMapper = new ObjectMapper();
+    @Autowired
```

Comment:
inline annotations looks clearer to me, but I guess it's subjective

Reply:
it has been changed by the intellij auto-formatter and tbh I prefer it this way